js/views/itemShortVw.js

```diff
@@ -41,7 +43,23 @@ module.exports = Backbone.View.extend({
       }
 
       this.model.set('ownFollowing', false);
-    });    
+    });
+
+    this.listenTo(window.obEventBus, "blockingUser", function(e){
+      if (e.guid !== this.model.get('guid')) {
+        return;
+      }
+
+      this.model.set('isBlocked', true);
```

Comment:
Six of one, or half a dozen of the other :)

The main reason was that if the functionality of this function would become lengthy, in the other approach it would all be indented. So, this way saves a level of indentation.

It does probably look a little counter-intuitive with the subsequent logic being so short. I'd have no objection to changing it, if you want?